packages/react-ui-ag/src/Listings/MobileListing.js

```diff
@@ -41,14 +39,19 @@ export default class MobileListing extends PureComponent {
   }
 
   get renderTopComponents() {
-    const { theme, listing, ratings, sponsoredText, categoryMatch, noMatchText } = this.props
+    const { theme, listing, ratings, noMatchText } = this.props
     const { singleFamily, rating, sponsored } = listing
-    return !singleFamily && (
-      <div className={theme.MobileListing_RatingAndSponsored}>
-        {!categoryMatch && <div className={theme.MobileListing_NoMatch}>{noMatchText}</div>}
-        {rating && <ListingComponents.Ratings data-tid="ratings" {...ratings} />}
-        {sponsored && <Text className={theme.MobileListing_Sponsored}>{sponsoredText}</Text>}
-      </div>
+
+    return (
+      <React.Fragment>
+        {noMatchText && <Text className={theme.MobileListing_NoMatchText}>{noMatchText}</Text>}
+        {!singleFamily &&
+          <div className={theme.MobileListing_RatingAndSponsored}>
```

Comment:
I hate to add more conditionals here, but it looks like the `MobileListing_RatingAndSponsored` div is going to be rendered regardless of either of its children existing. I might be wrong, but from what I've seen its fairly common for neither to exist, and we could shave a off a DOM node.